README.md

```diff
@@ -20,12 +20,16 @@ The easiest and quickest way to get started with Analytics 2.0 is to [use it thr
 
 ## 💡 Using with Segment
 
+**There is a [React example repo](https://github.com/segmentio/react-example/tree/main/src/examples/analytics-quick-start) which outlines using the Segment snippet.**
```

Comment:
The links out to these examples are well documented; developers should be able to read it along with going through the repo and see how to implement the examples.